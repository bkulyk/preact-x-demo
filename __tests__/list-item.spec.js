import React from 'react';
import ListItem from '@bitchin/list-item';
import { shallow } from './enzyme';
import { it, expect } from './jest';

it('should render correctly', () => {
  expect(shallow(<ListItem />)).toMatchSnapshot();
});

it('should render be able to render a different component', () => {
  expect(shallow(<ListItem component="dd" />)).toMatchSnapshot();
});

it('should render be able to render with props', () => {
  expect(shallow((
    <ListItem data-alliance="Rebel" className="pilot">Han Solo</ListItem>
  ))).toMatchSnapshot();
});
