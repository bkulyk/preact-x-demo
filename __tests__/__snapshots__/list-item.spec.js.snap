// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render be able to render a different component 1`] = `
<dd
  className="mdc-list-item"
/>
`;

exports[`should render be able to render with props 1`] = `
<li
  className="mdc-list-item pilot"
  data-alliance="Rebel"
>
  Han Solo
</li>
`;

exports[`should render correctly 1`] = `
<li
  className="mdc-list-item"
/>
`;
